sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -440,6 +442,10 @@ def test_cos_rewrite():
     assert cos(x).rewrite(sin) == sin(x + pi/2, evaluate=False)
     assert cos(x).rewrite(csc) == 1/csc(-x + pi/2, evaluate=False)
     assert cos(sin(x)).rewrite(Pow) == cos(sin(x))
+    assert cos(x).rewrite(besselj) == Piecewise((sqrt(2)*sqrt(pi)*sqrt(x)*besselj(-S.Half, x)/2, Ne(x, 0)), (1, True))
+    assert cos(cosh(x)).rewrite(besselj) == \
+        Piecewise((sqrt(2)*sqrt(pi)*sqrt(cosh(x))*besselj(-S.Half, cosh(x))/2, Ne(cosh(x), 0)), (1, True))
```

Comment:
Long lines should be broken e.g.:
```
assert cos(x).rewrite(besselj) == Piecewise(
    (sqrt(2)*sqrt(pi)*sqrt(x)*besselj(-S.Half, x)/2, Ne(x, 0)),
    (1, True)
)
```
I don't see what the purpose of the `cos(cosh(x)).rewrite(besselj)` test is.

Reply:
The point about long lines applies to to all changes in the PR.